src/table/TableBodyWrapper.jsx

```diff
@@ -57,18 +90,43 @@ const TableBodyWrapper = ({
     colIdx: -1,
     isEnabled: selectionsEnabled,
   });
+  const [isScrollbar, setIsScrollbar] = useState(false);
 
   useEffect(() => {
     selDispatch({ type: 'set-enabled', payload: { isEnabled: selectionsEnabled } });
     setColumnRenderers(getColumnRenderers);
   }, [selectionsEnabled, columns.length]);
 
+  // Setup isScrolling variable
+  let isScrolling;
+  // eslint-disable-next-line consistent-return
```

Comment:
Bad idea. When you add more stuff here that `addSelectionListeners` I think that those listeners wont unmount.

Reply:
Looking into it. Not sure how to fix it now.

Reply:
fixed